Apps/Sandcastle/CesiumSandcastle.js

```diff
@@ -768,6 +768,23 @@ require({
                         appendConsole('consoleError', 'Unable to GET from GitHub API. This could be due to too many request, try again in an hour or copy and paste the code from the gist: https://gist.github.com/' + gistId , true);
                         console.log(error);
                 });
+            } else if (defined(queryObject.code)) {
+                //The code query parameter is a Base64 encoded JSON string with `code` and `html` properties.
+                var json = JSON.parse(window.atob(queryObject.code));
+                var code = json.code;
+                var html = json.html;
+
+                jsEditor.setValue(code);
+                htmlEditor.setValue(html);
+                demoCode = code.replace(/\s/g, '');
+                demoHtml = html.replace(/\s/g, '');
+                gistCode = code;
+                gistHtml = html;
+                previousCode = code;
+                previousHtml = html;
+                sandcastleUrl = getBaseUri(window.location.href) + '?src=Hello%20World.html&label=Showcases&code=' + code;
```

Comment:
Out of scope for this PR, but it's always bugged me that Sandcastle still needs `src` and `label` even when they don't apply at all.

Reply:
Yeah, I noticed that but didn't want to get carried away, this is a very targeted PR that tries to not mess with anything else.  Sandcastle is about 2 years overdue for a rewrite.